### DISCUSSION TITLE
Deleting an annotation

### DISCUSSION THREAD

**Discussion by mariomui** [aidenlx/obsidian-zotero#148] [Help] [answered]
When I delete a highlight on Zotero, the vault lit note does not delete. Is it just me? I reset the settings to vanilla but the unexpected behavior (ub) persists.

**Comment by aidenlx** [comment, marked as answer]
Should be fixed in latest version now, leave more comment if you still have trouble with it

**Comment by starise** [comment]
I have the same issue with the latest version (1.1.7) and Zotero 7.
